warehouse/email/ses/models.py

```diff
@@ -262,22 +264,21 @@ class EventTypes(enum.Enum):
 class Event(db.Model):
     __tablename__ = "ses_events"
 
-    created = mapped_column(DateTime, nullable=False, server_default=sql.func.now())
+    created: Mapped[datetime] = mapped_column(DateTime, server_default=sql.func.now())
 
-    email_id = mapped_column(
-        UUID(as_uuid=True),
+    email_id: Mapped[UUID] = mapped_column(
+        PG_UUID(as_uuid=True),
         ForeignKey(
             "ses_emails.id", deferrable=True, initially="DEFERRED", ondelete="CASCADE"
         ),
-        nullable=False,
         index=True,
     )
 
-    event_id = mapped_column(Text, nullable=False, unique=True, index=True)
-    event_type = mapped_column(
-        Enum(EventTypes, values_callable=lambda x: [e.value for e in x]), nullable=False
+    event_id: Mapped[str] = mapped_column(Text, unique=True, index=True)
+    event_type: Mapped[Enum] = mapped_column(
+        Enum(EventTypes, values_callable=lambda x: [e.value for e in x])
     )
 
-    data = mapped_column(
-        MutableDict.as_mutable(JSONB), nullable=False, server_default=sql.text("'{}'")  # type: ignore[arg-type] # noqa: E501
+    data: Mapped[dict] = mapped_column(
+        MutableDict.as_mutable(JSONB), server_default=sql.text("'{}'")  # type: ignore[arg-type] # noqa: E501
```

Comment:
I _think_ we can remove the `type: ignore` now - would you mind trying it locally, and removing it if no longer needed?

Reply:
```warehouse/email/ses/models.py:283: error: Argument 1 to "as_mutable" of "Mutable" has incompatible type "type[JSONB]"; expected "TypeEngine[Any]"  [arg-type]
```

Reply:
Okay, thanks! We'll come back to that someday, I'm sure.